tests/main.test.ts

```diff
@@ -17,8 +17,9 @@ type Sender = Context["payload"]["sender"];
 
 const octokit = jest.requireActual("@octokit/rest");
 
-const url = process.env.SUPABASE_URL;
-const key = process.env.SUPABASE_KEY;
+const url = "https://sbxscxthunnhozvgtshx.supabase.co";
```

Comment:
We should not use real endpoints. If it is to put dummy values, then set something that doesn't exists. If it is to actually reach a db, you have 2 options:

- mock Supabase library (efficient but tedious)
- intercept requests with `msw` (longer to do but easier to use on the long run from my experience)